authz/providers/azure/rbac/rbac.go

```diff
@@ -504,10 +506,16 @@ func (a *AccessInfo) sendCheckAccessRequest(ctx context.Context, checkAccessUser
 		checkAccessSucceeded.Inc()
 	}
 
-	// Decode response and prepare k8s response
-	status, err := ConvertCheckAccessResponse(checkAccessUsername, data)
-	if err != nil {
-		return err
+	var status *authzv1.SubjectAccessReviewStatus
+	if isTrackedResource && resp.StatusCode == http.StatusNotFound {
```

Comment:
According to my experiments, StatusNotFound can only be returned for Tracked child resources. Top level resource cannot be 'ProxyOnly'. so isTrackedResource is not needed. See my previous comment for more details.